### PR TITLE
[PDI-19865] - Job fails if it contains 2 consecutive Transformation job entries writing to the same logfile when the Logfile Name is specified by UNC path

### PR DESCRIPTION
**Pull request by renato-s**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 49m 21s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **5324**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9014/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java [hunk 1]
```diff
@@ -413,8 +414,10 @@ public static String getFilename( FileObject fileObject ) {
     if ( !root.startsWith( "file:" ) ) {
       return fileName.getURI(); // nothing we can do about non-normal files.
     }
+    // PDI-PDI-19865 - we'll see 4 forward slashes for a windows/smb network share,
```
Comment (bcostahitachivantara) [inline core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java:417]: Just a detail: PDI-PDI-19865 double Jira Project ID

**Review by bcostahitachivantara — APPROVED** [review]

File: core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java (hunk 1 above)
Comment (renato-s) [reply core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java:417]: Fixed

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 50m 51s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **5324**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9014/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**